qcfractal/tests/test_collections.py

```diff
@@ -159,6 +159,35 @@ def test_dataset_compute_response(fractal_compute_server):
     assert len(response.ids) == 2
 
 
+def test_dataset_d3(fractal_compute_server):
+    testing.check_has_module("psi4")
```

Comment:
`testing.check_has_module("psi4")` is only needed for fixtures. Can you use the current -d3 dataset to test this out rather than building a new one?